UnitsNet.Tests/QuantityIFormattableTests.cs

```diff
@@ -46,12 +46,12 @@ public void QFormatEqualsQuantityName()
         [Fact]
         public void SFormatEqualsSignificantDigits()
         {
-            Assert.Equal(length.ToString(null, 0), length.ToString("s"));
-            Assert.Equal(length.ToString(null, 1), length.ToString("s1"));
-            Assert.Equal(length.ToString(null, 2), length.ToString("s2"));
-            Assert.Equal(length.ToString(null, 3), length.ToString("s3"));
-            Assert.Equal(length.ToString(null, 4), length.ToString("s4"));
-            Assert.Equal(length.ToString(null, 5), length.ToString("s5"));
+            Assert.Equal("1 ft", length.ToString("s"));
```

Comment:
Much better to explicitly state the values 👍 
But isn't this test a duplicate of `CustomNumberOfSignificantDigitsAfterRadixFormatting` ? I like this test method name `SFormatEqualsSignificantDigits` better though, if you merge the two.

Reply:
Replaced redundant test and made data-driven.